tests/scripts/run-scan-api.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+ENV_FILE=${1}
+export $(grep -v '^#' $ENV_FILE | xargs)
```

Comment:
This script should be improved – more info in the comments for package.json

Reply:
done